libraries/lib-utility/MemoryX.h

```diff
@@ -601,7 +601,7 @@ NonInterferingBase {
    static void *operator new(std::size_t count, std::align_val_t al);
    static void operator delete(void *ptr, std::align_val_t al);
 
-#if defined (_MSC_VER) && defined(_DEBUG)
+#if defined(_MSC_VER) && defined(_DEBUG) && !defined(HAVE_LIB_TORCH)
```

Comment:
Why was this change necessary to your build?

Reply:
Now I see an explanation in AudacityHeaders.h.  Please repeat it here.

Reply:
resolved in 3b4f57d5c